themes/src/main/resources/theme/keycloak-preview/account/index.ftl

```diff
@@ -45,7 +45,8 @@
                 isLinkedAccountsEnabled : ${realm.identityFederationEnabled?c},
                 isEventsEnabled : ${isEventsEnabled?c},
                 isMyResourcesEnabled : ${(realm.userManagedAccessAllowed && isAuthorizationEnabled)?c},
-                isTotpConfigured : ${isTotpConfigured?c}
+                isTotpConfigured : ${isTotpConfigured?c},
+                deleteAccountAllowed : ${deleteAccountAllowed?c}
```

Comment:
Enabling/disabling the delete account should not be done here. Rather, it should be done with the JavaScript code to check if the authenticated user has the delete-account role.

Reply:
@stianst I'm ok with this as long as the proper check is done on the server as the page is being rendered.  Is there something that would be different on the client side?

Reply:
@stianst can you please explain the motives behind shifting the logic to the client-side ? to be able to do check from the front end, we should also modify the `AccountRestService` to return to user roles, we will end up doing the same thing, kind of.